src/windows/region.py

```diff
@@ -133,9 +130,6 @@ def __init__(self, file=None, clip=None):
         self.r.SetMaxSize(self.viewport_rect.width(), self.viewport_rect.height())
 
         try:
-            # Add clip for current preview file
-            self.clip = openshot.Clip(self.file_path)
```

Comment:
This was recreating the attached Clip object for no reason (at least no reason I could tell). By removing this line, we can still access the `self.clip` object, which already has the Start() and End() properties set. However, we then add this clip to a new Timeline at `position: 0.0`. So our slider always starts at frame 1 and ends at the `clip.Duration() * clip.fps()`.